transmart/api/v2/query.py

```diff
@@ -62,6 +77,21 @@ def __repr__(self):
                                                                  self.method,
                                                                  self.params)
 
+    def set_concept_constraint(self, in_concept):
+        if isinstance(in_concept, list):
+            self.in_concept = []
+            for concept_constraint in in_concept:
+                if "\\" in concept_constraint:
```

Comment:
its better to have the user choose between code and path in the python api instead of guessing here.